build.sh

```diff
@@ -170,7 +190,7 @@ cmd_pack() {
     dotnet pack --no-restore --no-build --nologo -o $OUTDIR \
            $SCRIPT_DIR/build/pack.csproj \
            /p:Version=$VERSION \
-           /p:NuspecFile=$(readlink -f $nuspec)
+           /p:NuspecFile=$(realpath $nuspec)
```

Comment:
Need `dash` here also. 
```bash
  -p:Version=$VERSION \
  -p:NuspecFile=$(realpath $nuspec)
```

Reply:
Oh, sorry about that, thanks for noticing